--- Makefile	2021-03-11 05:27:51.000000000 +0000
+++ Makefile.openssl1.1.1j	2021-03-11 06:00:56.000000000 +0000
@@ -842,7 +842,7 @@
 	objs/addon/src/ngx_stream_lua_logby.o \
 	objs/addon/src/ngx_stream_lua_prereadby.o \
 	objs/ngx_modules.o \
-	-L/tmp/openresty-1.13.6.2/build/luajit-root/opt/ibm/router/luajit/lib -L/tmp/openresty-1.13.6.2/build/luajit-root/opt/ibm/router/luajit/lib -Wl,-rpath,/opt/ibm/router/luajit/lib -Wl,-E -Wl,-E -ldl -lpthread -lpthread -lcrypt -L/tmp/openresty-1.13.6.2/build/luajit-root/opt/ibm/router/luajit/lib -lluajit-5.1 -lm -ldl -L/tmp/openresty-1.13.6.2/build/luajit-root/opt/ibm/router/luajit/lib -lluajit-5.1 -lm -ldl /tmp/pcre-8.42/.libs/libpcre.a /tmp/openssl-1.1.1j/.openssl/lib/libssl.a /tmp/openssl-1.1.1j/.openssl/lib/libcrypto.a -ldl -lz \
+	-L/tmp/openresty-1.13.6.2/build/luajit-root/opt/ibm/router/luajit/lib -L/tmp/openresty-1.13.6.2/build/luajit-root/opt/ibm/router/luajit/lib -Wl,-rpath,/opt/ibm/router/luajit/lib -Wl,-E -Wl,-E -ldl -lpthread -lcrypt -L/tmp/openresty-1.13.6.2/build/luajit-root/opt/ibm/router/luajit/lib -lluajit-5.1 -lm -ldl -L/tmp/openresty-1.13.6.2/build/luajit-root/opt/ibm/router/luajit/lib -lluajit-5.1 -lm -ldl /tmp/pcre-8.42/.libs/libpcre.a /tmp/openssl-1.1.1j/.openssl/lib/libssl.a /tmp/openssl-1.1.1j/.openssl/lib/libcrypto.a -ldl -lz -lpthread \
 	-Wl,-E
 	
 
